app/models/feed_event.rb

```diff
@@ -42,4 +47,54 @@ def self.record_journey_for(user, article:, category:)
         article: article,
       )
   end
+
+  def self.bulk_update_counters_by_article_id(article_ids)
+    unique_article_ids = article_ids.uniq
+    update_counters_for_articles(unique_article_ids)
+  end
+
+  def self.update_counters_for_articles(article_ids)
+    article_ids.each do |article_id|
+      update_single_article_counters(article_id)
+    end
+  end
+
+  def self.update_single_article_counters(article_id)
+    ThrottledCall.perform("billboards_data_update-#{article_id}", throttle_for: 15.minutes) do
```

Comment:
```suggestion
    ThrottledCall.perform(""article_feed_success_score_#{article_id}"", throttle_for: 15.minutes) do
```

Reply:
@filleduchaos I think you accidentally included more double quotes here

Reply:
😅 ah yes, thanks!